### DISCUSSION TITLE
Audio Transcription - ffmpeg not found

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#123] [Q&A] [answered]
Hi, I am testing de audio transcription feature, with Google Service implementation, but I am having the following problem:

_java.io.IOException: Cannot run program "ffmpeg"
Error testing ffmpeg, that could hurt transcription. Is it on path?
FFmpeg not detected, audios longer than 1min will not be transcribed!_

I downloaded ffmpeg, added .exe as system variable, but it didn't work. need some help

**Comment by lfcnassif** [comment, marked as answer]
You need to add parent folder of ffmpeg.exe in PATH environment variable. Did you add the .exe itself?

**Comment by gisms** [comment]
yes, now it worked! thanks

I used this two lines in my .bat to execute iped:

set GOOGLE_APPLICATION_CREDENTIALS=[path to project .json]
set ffmpeg=[path ffmpeg\bin\ ]

**Comment by lfcnassif** [comment]
Great! Thanks for the feedback. Note: just google impl needs ffmpeg, microsoft doesn't need it. But results are not equal, of course.

**Reply by lfcnassif** [reply]
Just to let you know, in 4.1.1 ffmpeg is not needed anymore.
